package.json

```diff
@@ -97,7 +101,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged && npm run test:unit && npm run test:e2e:dev",
+      "pre-commit": "lint-staged -r && npm run test:unit && npm run test:e2e:dev",
```

Comment:
What's the `-r` for?

Reply:
Its to tell lint-staged to pass a relative path to tasks. I was facing a bug initially so I tried that. I was meant to remove it